### ISSUE TITLE
net-im/telegram-0.9.33 fails to merge

### ISSUE THREAD

**Issue by p-kozlov** [rindeal/gentoo-overlay#16]
```
In file included from /usr/include/breakpad/client/linux/minidump_writer/minidump_writer.h:41:0,
                 from /usr/include/breakpad/client/linux/handler/exception_handler.h:42,
                 from ../../Telegram/SourceFiles/logs.cpp:42:
/usr/include/breakpad/client/linux/minidump_writer/linux_dumper.h:50:58: fatal error: client/linux/dump_writer_common/mapping_info.h: No such file or directory
 #include "client/linux/dump_writer_common/mapping_info.h"
                                                          ^
compilation terminated.
Makefile:19751: recipe for target '../ReleaseIntermediate/logs.o' failed
make: *** [../ReleaseIntermediate/logs.o] Error 1
make: *** Waiting for unfinished jobs....
 * ERROR: net-im/telegram-0.9.33::rindeal failed (compile phase):
 *   emake failed
 * 
```

**Comment by rindeal** [comment]
Please attach full build log and the output of `equery f dev-util/google-breakpad` command.

**Comment by p-kozlov** [comment]
breakpad files: https://bpaste.net/show/d6751c7be7b5
full build logs: https://bpaste.net/show/68dee2e115b6

**Comment by rindeal** [comment]
You're using some weird/outdated version of breakpad. Reemerge it (`emerge -1 dev-util/google-breakpad::rindeal`) and try again.

[Here is my file list](https://github.com/rindeal/gentoo-overlay/files/200885/google-breakpad_files.txt)  for a reference. And [here is yours](https://github.com/rindeal/gentoo-overlay/files/200886/p-kozlov-google-breakpad_files.txt) for a future reference as your bpaste soon expires.

**Comment by rindeal** [comment]
If that won't help reopen the issue.

**Comment by p-kozlov** [comment]
@rindeal well, that is strange, I merged dev-util/google-breakpad just 31.03.2016, but can try to update it again. Will see if it helps.

**Comment by rindeal** [comment]
> I decreased priority from 50 to 40 for zugaina overlay - hope this will help.

A better solution would be to mask the package like this:

**/etc/portage/package.mask**:
```
dev-util/google-breakpad::zugaina
```


Also, I plan to remove support for it some day, beginning with #17, to prevent issues like this.

---

P.S. @p-kozlov please, do not use "Edit comment" for major updates like this as no notifications are issued for such actions.
